src/openloco/Window.h

```diff
@@ -156,13 +157,13 @@ namespace openloco::ui
             struct
             {
                 uint32_t on_close;
-                void (*on_mouse_up)(window*, widget_index);
+                void (*onClick)(Window*, widget_index);
```

Comment:
Hmm, while I can see the logic behind calling it a click event, I think the having contrasting MouseUp and MouseDown events is the better naming choice — because which one is click, to someone new to the code base?